admin/links/class-link-content-processor.php

```diff
@@ -34,6 +34,10 @@ public function __construct( WPSEO_Link_Storage $storage, WPSEO_Meta_Storage $co
 	 * @param string $content The content to process.
 	 */
 	public function process( $post_id, $content ) {
+		if ( ! $this->storage->table_exists() ) {
```

Comment:
Can we use the `is accessible` functionality from the storage class here?

Reply:
I'll use the `check_if_table` exists function.